lang/tr.js

```diff
@@ -181,102 +182,102 @@ var theUILang =
- Address			: "Address",
- PeerDL				: "Peer DL",
- PeerDownloaded			: "Peer Downloaded",
- Save_path			: "Save path",
- registerMagnet			: "Try to register magnet protocol handler at start",
- linkTorTorrentRestored		: "Connection to rTorrent established.",
- removeAllTegs			: "Remove all tags",
- Torrents			: "Torrent(s)",
- copyToClipboardFailed		: "The copy function isn't working in your browser.\nPlease copy this content manually:\n\n",
- copyToClipboardSuccess		: "Copied to clipboard successfully!"
+ updateTracker			: "İzleyicileri Güncelle",
+ scrapeUpdate			: "Son Güncelleme",
+ trkInterval			: "Sıklık",
+ logAutoSwitch			: "'Günlük' sekmesine otomatik geç",
+ showLabelSize			: "Etiket boyutunu göster",
+ phpParameterUnavailable	: "PHP yönergesi register_argc_argv OFF olarak ayarlandı. ON olarak değiştirin, aksi halde bazı eklentiler düzgün çalışmayacaktır.",
+ addTorrentFailedURL : "Torrent eklenemedi. URL alınamıyor.",
+ addTorrentFailedFile : "Torrent eklenemedi. Alınan içerik geçerli bir torrent dosyası değil.",
+ addTorrentFailedDirectory : "Torrent eklenemedi. Hatalı indirme dizini belirtildi.",
+ plgLaunch			: "Başlat",
+ plgLocked			: "Zorunlu",
+ phpExtensionNotFoundWarning	: "Bazı işlevler kullanılamayacak. PHP uzantıları kurulmalı",
+ phpExtensionNotFoundError	: "Eklenti çalışmayacak. PHP uzantıları kurulmalı",
+ trkPrivate			: "Özel",
+ dontSet			: "Ayarlama",
+ webBadSessionError		: "Eklenti çalışmayacak. Web sunucusu kullanıcısı rtorrent oturum dizinine yürütme erişimine sahip olmalıdır",
+ webBadSessionWarning		: "Bazı işlevler kullanılamayacak. Web sunucusu kullanıcısı rtorrent oturum dizinine yürütme erişimine sahip olmalıdır",
+ Developers			: "Geliştiriciler",
+ Check_new_version		: "Yeni sürümü kontrol et",
+ here				: "burada",
+ Num_No				: "Nu.", // This means "Number". To be used as header for number count, e.g.: throttle/ratio plugin
+ This_screen			: "Bu ekran",
+ About_program			: "ruTorrent Hakkında",
+ Toggle_menu			: "Menüyü değiştir",
+ Toggle_details			: "Ayrıntıları değiştir",
+ Toggle_categories		: "Kategorileri değiştir",
+ Delete_current_torrents	: "Geçerli torrent(ler)i sil",
+ Select_all : "Tümünü seç",
+ Deselect_all : "Tümünün seçimini kaldır",
+ showSpeedInTitle : "Başlıkta hızı göster",
+ badTempPath : "Web sunucusu kullanıcısının tmp dizinine okuma/yazma/yürütme erişimi yok. ruTorrent çalışmayacak.",
+ badTempPath2 : "rTorrent kullanıcısının tmp dizinine okuma/yürütme erişimi olmalıdır. ruTorrent çalışmayacak.",
+ doRandomizeHash : "Çapraz seedleme için info hashleri rastgele düzenle",
+ Recreate_files : "Sıradaki Dosyaları Yeniden Oluşturma",
+ Address			: "Adres",
+ PeerDL				: "Eş DL",
+ PeerDownloaded			: "İndirmiş Eşler",
+ Save_path : "Yolu kaydet",
+ registerMagnet : "Başlangıçta mıknatıs protokolü işleyicisini kaydetmeyi deneyin",
+ linkTorTorrentRestored : "rTorrent ile bağlantı kuruldu.",
+ removeAllTegs : "Tüm etiketleri kaldır",
+ Torrentler: "Torrent(ler)",
```

Comment:
This seems wrong - renamed the key instead of just the string.

Reply:
Fixed, thanks.